src/applications/discharge-wizard/components/v2/RequestDD214.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import {
+  determineBoardObj,
+  determineVenueAddress,
+  determineBranchOfService,
+} from '../../helpers';
+import { SHORT_NAME_MAP } from '../../constants/question-data-map';
+import { pageSetup } from '../../utilities/page-setup';
+
+const RequestDD214v2 = ({ router, formResponses, viewedIntroPage }) => {
+  const H1 =
+    'Your Steps for Getting a DD214 for Your Period of Honorable Service';
+
+  useEffect(
+    () => {
+      pageSetup(H1);
+    },
+    [H1],
+  );
+
+  useEffect(() => {
+    if (!viewedIntroPage) {
+      router.push('/');
+    }
+  });
+
+  const { name, abbr } = determineBoardObj(formResponses, true);
+  const branchOfService = determineBranchOfService(
+    formResponses[SHORT_NAME_MAP.SERVICE_BRANCH],
+  );
+  return (
+    <div>
+      <h1>{H1}</h1>
+      <div className="">
+        <div className="va-introtext">
+          <p>
+            To receive a second DD214 reflecting <strong>only</strong> your
+            period of honorable service, you’ll need to complete Department of
+            Defense (DoD) Form 149 and send it to the {name}—
+            <strong>do not</strong> send it to the Discharge Review Board (DRB)
+            for the {branchOfService}.
+          </p>
+        </div>
+        <va-process-list>
+          <va-process-list-item>
+            <h2 className="usa-process-list__heading">
+              Download and fill out{' '}
+              <va-link
```

Comment:
(Even though this text is giant) Does this need to say "(opens in a new tab)"?

Reply:
@randimays This doesn't actually open in a new tab though - at least it didn't for me.

I think if we were to do this, I would prefer to change this format and take the link out of the header, have it formatted more like the previous page - Header, tips for filling out the form, download link at the bottom. But is that a question for CAIA if we can redo this?

Reply:
Oops, I forgot to mention that perhaps it should open in a new tab since it's an external link. Just curious - don't have an opinion either way. Perhaps need a Product/Design call.

Reply:
I would also prefer to take the link out of the header. 

@FranECross Do we need to ask CAIA if we can do what Laura is suggesting?

Reply:
This did not open in a new tab for me. Goes directly to the form for download. I think thats a good question for CAIA to look at page formatting of information.

Reply:
> I would also prefer to take the link out of the header.
> 
> @FranECross Do we need to ask CAIA if we can do what Laura is suggesting?

I don't think we need to check with CAIA, and we can emulate the design that is in place on the results screen, yes?

![image](https://github.com/user-attachments/assets/c7b16bfb-e51e-48f6-8e8f-79329765aa70)

Reply:
Updated with latest push!

![How_To_Apply_For_A_Discharge_Upgrade___Veterans_Affairs](https://github.com/user-attachments/assets/cd740b21-8c97-4f44-8f9d-383cd93f0c4b)

Reply:
@chriskim2311 Not sure if this review instance will be up by the time I sign off but this screen shot looks good, thanks!

Reply:
@chriskim2311 LGTM!